package.json

```diff
@@ -22,7 +22,7 @@
     "chai-as-promised": "^7.1.1",
     "ganache-cli": "^6.0.3",
     "mocha": "^4.0.1",
-    "ts-node": "^3.3.0",
+    "ts-node": "3.3.0",
```

Comment:
is there any particular reason for updating this dependency?

Reply:
It makes harder to review changes (lots of package.lock changes) so I prefer to do it in separate PR.